Automattic/jetpack
# Add support for SIG OpenGraph tags

This change makes the plugin use the SIG image for the OpenGraph tags, if it is enabled. Right know it's handled in two different places, but I used a common hook for the logic.

### Other information:


## Jetpack product discussion



1203876111187910-as-1203968442376755

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






Before the new Social Image Generator service is merged, it can be only tested by abtracting that.

In `projects/packages/publicize/src/class-publicize-base.php` comment out `$sig       = new Social_Image_Generator();` in the 1499. line, and at it's usage below just provide a dummy string.

Do the following with Social standalone, then Jetpack:
* Create a new post and add a featured image. In the SIG settings disable SIG and publish the post ( no need to share it ). Open the created blogpost, and in the inspector search for the `og:image` tag. It should point to the featured image.
* Go back to the editor and enable SIG and update the post.
* Refresh the opened blogpost, now the `og:image` should contain the dummy string you specified before
